rollup.config.js

```diff
@@ -136,6 +136,7 @@ const stripOptions = {
         'Debug.warn',
         'Debug.error',
         'Debug.log',
+        'Debug.trace',
```

Comment:
Is possible (/does it make sense) to remove `setTrace` and `_traceChannels` too?

Reply:
_traceChannels is automatically stripped (the same as the existing other set).
setTrace is an empty function in non debug build which I'd like to keep in .. allowing the code to compile in non-debug builds as well.

Reply:
Wouldn't the rollup plugin remove references in the code to 'Debug.setTrace' as well as the function itself?

Reply:
Isn't that an issue for `Debug.trace` anyway?

Reply:
well, Debug.trace is to be used inside the engine only .. but Debug.setTrace is called from outside to enabled the tracing. That's my plan on how this should be used.

Reply:
I see, makes sense 👍